meta-freescale-mbl/conf/layer.conf

```diff
@@ -10,12 +10,7 @@ BBFILES += "${LAYERDIR}/recipes*/*/*.bb ${LAYERDIR}/recipes*/*/*.bbappend"
 
 BBFILE_COLLECTIONS += "freescale-layer-mbl"
 BBFILE_PATTERN_freescale-layer-mbl := "^${LAYERDIR}/"
-BBFILE_PRIORITY_freescale-layer-mbl = "6"
 
-LAYERSERIES_COMPAT_freescale-layer-mbl = "sumo thud warrior"
+LAYERSERIES_COMPAT_freescale-layer-mbl = "warrior"
 LAYERDEPENDS_freescale-layer-mbl = "freescale-layer"
 LAYERDEPENDS_freescale-layer-mbl += "meta-mbl-bsp-common"
-
-# This layer depends on the openembedded-core/meta u-boot-tools*.bb
-# recipe (for example), so explicly include the dependency. 
-LAYERDEPENDS_freescale-layer-mbl += "core-mbl"
```

Comment:
> All meta-layers are inherently dependent on core whether its explicit or not, so maybe its OK to remove the LAYERDEPENDS.

I think that we should rely on this for all layers except the `openembedded-core-mbl` which we want to have a higher priority.

Reply:
Conversation:

Simon Hughes [3:28 PM]
re:
>> openembedded-core-mbl which we want to have a higher priority.
why do we want the staging layer to have a higher priority than the upstream?

Diego Sueiro [3:30 PM]
Because we want override stuff from the upstream. This is what we discussed this morning. And by using the LAYERDEPENDS bitbake sets the priority higher automatically.

Simon Hughes [3:35 PM]
yes, but afaik there are no openembedded-core-mbl recipes that overlay openembedded-core recipes at this time
unless I've missed something
checking for overlays ...

Simon Hughes [3:51 PM]
gstreamer1.0-omx_1.12%.bb in openembedded-core-mbl overlays gstreamer1.0-omx_1.14.4.bb in openembedded-core
(this suggests we should uprev to 1.14.4)

Diego Sueiro [3:52 PM]
I'm now raising a jira to remove and move some recipes around. And this one is included.

Simon Hughes [3:53 PM]
ok
where would you move it to?

Diego Sueiro [3:53 PM]
I think this one should be removed.

Simon Hughes [3:54 PM]
yes

Diego Sueiro [3:54 PM]
As the avahi
Some bbappends are distro specific. I'm still looking into it.
As soon as I create the Jira I'll send you the link and you can review the proposal.

Simon Hughes [3:55 PM]
OK. what do you want to do about the openembedded-core-mbl LAYERDEPENDS on core?

Diego Sueiro [3:56 PM]
I want to leave it there.

Simon Hughes [3:56 PM]
forever, or to remove it later when the gstreamer recipe is removed, or something else?

Diego Sueiro [3:57 PM]
Forever. It is a staging layer and I think we should explicitly set the dependency on the upstream.

Simon Hughes [3:58 PM]
thinking...

Simon Hughes [4:06 PM]
ok, I'm undecided/think its marginal so I'm OK to go with whats in PR547

Reply:
@diego-sueiro Please could you put the jira ticket number mentioned above on the ticket? Thanks.